workshop/SCHEDULE.md

```diff
@@ -15,7 +15,7 @@ While we expect to start and end each day on time, the timing of individual modu
 |-------------|------------------------------------------------|----------|
 | **Day 1**   | **2021-06-28** <br> [_**Introduction to R and the Tidyverse**_](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/README.md)
 | 12:00 PM    | Welcome, Introductions and Getting Started  <br> [Workshop Introduction slides (pdf)](../slides/2021-06-28_CCDL_workshop_intro.pdf)  | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
-| 1:00 PM     | [Introduction to base R](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/01-intro_to_base_R.nb.html) <br> | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
+| 1:00 PM     | [Introduction to Rstudio Server (pdf)](../slides/2021-06-28_Intro_to_Rstudio.pdf) <br> [Introduction to base R](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/01-intro_to_base_R.nb.html) | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
```

Comment:
```suggestion
| 1:00 PM     | [Introduction to Rstudio Server (pdf)](../slides/2021-06-28_Intro_to_R.pdf) <br> [Introduction to base R](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/01-intro_to_base_R.nb.html) | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
```

This would explain why think link isn't working for me -- I think this should fix it

Reply:
I renamed the file, as I had intended, rather than rewriting the link. Can you check that it works now?

Reply:
Yes, it now works 🎉